### DISCUSSION TITLE
How does PaddleOCR performance compare to Tesseract?

### DISCUSSION THREAD

**Discussion by PPGitub** [PaddlePaddle/PaddleOCR#8349] [Q&A]
How does PaddleOCR performance compare to Tesseract?

**Comment by MissPenguin** [comment]
Haven't compared with Tesseract recently. Here are some comparative evaluations made by external users for reference.

https://converter.app/blog/paddleocr-engine-example-and-benchmark
https://learnopencv.com/optical-character-recognition-using-paddleocr/

**Comment by Evezerest** [comment]
the OCR model used in the learnopencv tutorial is the Chinese-English one. Generally, for pure English scenarios, it is recommended to use the English model ('lang='en') 
reference: https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/whl_en.md#21-use-by-code
